package.json

```diff
@@ -76,6 +75,12 @@
         "path": "./themes/sasjs-theme-dark.json"
       }
     ],
+    "jsonValidation": [
+      {
+        "fileMatch": ".sasjslint",
+        "url": "https://raw.githubusercontent.com/sasjs/lint/main/sasjslint-schema.json"
+      }
+    ],
```

Comment:
nice! does this work on any project?

Reply:
Ah, no. This is in the `contributes` section which is specific to VS Code extensions.
We can add other entries here though, so whoever is using our extension can use the associated validation.